_posts/2019-03-29-tmm-post.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "Trump Money Maker: Timing the Market"
+date:   2019-01-21 10:39:11 -0500
+categories: blog
+---
+_Major lessons learned during the developement of [Trump Money Maker]({% post_url 2019-03-16-tmm %})._
+![Trump Money Maker automated teller gif stamping envelopes]({{ "/assets/teller.gif" | relative_url }}){:class="float-right-plz teller-img"}
+<!-- Ends the excerpt text, it includes the image -->
+
+
+<br>
+
+
+Imagine you are 6 brite, eager, and adventuous roommates. Sam, RJ, and I lived with 3 of our friends in _The Mansion, RIP_. It was campaign season, early 2016, and we were very hot for politics and game dev. Obviously, not self controlled enough to separate those two feelings, they inevitably bled together into TMM.
```

Comment:
> brite

bright
> adventuous

adventurous

"The Mansion, RIP" is an inside joke, and may make our audience feel like they are not the appropriate audience

> Obviously, not self controlled enough to separate those two feelings, they inevitably bled together into TMM.
The implied subject of "not self controlled enough" is Sam, Aaron, and RJ, which disagrees with the subject of "they inevitably bled together into TMM

Reply:
👏 these are wonderful. I can't get my spell checking to work in markdown files and it's making me feel dumb!

Reply:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
wow…many many spelling errors 😬